config/initializers/node-orm2-connect.js

```diff
@@ -6,11 +6,16 @@ export default {
   before: 'define-orm-models',
   async initialize(application) {
     let container = application.container;
-    let adapter = container.lookup('orm-adapter:node-orm2');
-    let config = application.config.database;
+    let config = application.config;
+
+    if (!config.database || !config.database.orm2) {
+      // Config is not specified
+      return;
+    }
 
     try {
-      adapter.db = await fromNode((cb) => orm.connect(config.url, cb));
+      let connection = await fromNode((cb) => orm.connect(config.database.orm2, cb));
```

Comment:
Hmm. Not sure I like scoping the config inside `database` to `orm2`. It might be helpful if you have multiple adapters, but most apps will likely have just one.

What if we did something like `config.database.orm2 || config.database`? Multi-adapter apps could keep everything under `database` with no special rules for your "main" adapter and "other" adapters, and single-adapter apps could keep their config simple